packages/arb-token-bridge-ui/src/components/TransactionHistory/PendingTransactions.tsx

```diff
@@ -77,15 +77,15 @@ export const PendingTransactions = ({
 
       {/* Error loading transactions */}
       {error && (
-        <span className="flex gap-x-2 text-sm text-red-400">
+        <span className="flex gap-x-2 text-sm text-red-400 mx-2">
```

Comment:
No need for change in this file as the margin was only intended for notifications coming in the main-screen, not transaction history panel.